tests/integration/targets/azure_rm_sqlserver/tasks/main.yml

```
@@ -70,6 +117,20 @@
       - output.changed
       - output.state == 'Ready'
 
+- name: Gather facts SQL Server
+  azure_rm_sqlserver_info:
+    resource_group: "{{ resource_group }}"
+    server_name: "sqlsrv-extended-{{ random_postfix }}"
+  register: output
+- name: Debug
+  debug:
```

Use ansible.builtin.debug or ansible.legacy.debug instead
```suggestion
  ansible.builtin.debug:
```

---

Thank you.  I committed your suggestion

---

I removed the debug tasks